jest.config.js

```diff
@@ -49,6 +62,7 @@ module.exports = {
     ...packageTestConfigs,
     ...appTestConfigs,
     ...appPaths,
+    ...packagePaths,
```

Comment:
Will this change impact CRN?

Reply:
it will just make it so if a package has a custom jest config it will use that one instead